task_base.py

```diff
@@ -464,14 +475,25 @@ def wait(self):
                 sleeptime = max_sleep
             time.sleep(sleeptime)
 
+        if bool(self._failed_ee_tasks) is True:
+            num_failed_tasks = len(list(self._failed_ee_tasks.keys()))
+            raise EETaskError(ee_messages=self._failed_ee_tasks)
```

Comment:
In thinking about this, raising an exception means that the task will exit, right, which means we have no way of monitoring the status of any remaining ee tasks other than manually? Should we instead wait until all ee tasks are finished, and then report # failed, # succeeded, # cancelled, etc?

Reply:
this doesn't raise an exception until after the while loop, so all the queued EE tasks should be done